docs/content/primary-key-table/compaction.md

```diff
@@ -57,8 +57,8 @@ expect a mode to have maximum writing throughput, the compaction can be done slo
 You can use the following strategies for your table:
 
 ```shell
-num-sorted-run.stop-trigger = 2147483647
-sort-spill-threshold = 10
+num-sorted-run.stop-trigger = 10
+sort-spill-threshold = 2147483647
```

Comment:
I think:
If apply this config, no sorted-run will be spill when compaction or mergeOnRead, maybe cause OOM.
May I ask why you made this modification？

Reply:
Sorry,I made a mistake.I'll close it.